assets/javascripts/tests.js

```diff
@@ -120,9 +120,8 @@ function renderTimeAgo(data, type, row, position, notAvailableMessage) {
         return (haveData ?
             ('<span title="' + data + '">' + jQuery.timeago(data) + '</span>') :
             (notAvailableMessage ? notAvailableMessage : 'not yet'));
-    } else {
-        return haveData ? data : 0;
     }
+    return haveData ? data : 'Z-9999-12-31';
```

Comment:
a Y10K bug!!

Reply:
(you might be too young to know the reference ;)

Reply:
I actually know about this. But I guess in year 10 000 this is no longer relevant :-)

Reply:
that's what the cobol developers in the 60ties thought as well :)

Reply:
I could add an entry here: https://en.wikipedia.org/wiki/Timeline_of_the_far_future#Technological_projects :-)

Reply:
And according to the same wiki article "The Gregorian calendar will have drifted by about 10 days in relation to the seasons". So the calendar system would likely need to be adapted anyways.